ecs-logs-collector.sh

```diff
@@ -496,6 +499,51 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker "${info_system}"/docker/docker-sysconfig
+    ok
+  else
+    info "/etc/sysconfig/docker not found"
+  fi
+
+ try "Gathering Docker storage sysconfig"
+
+  if [ -e /etc/sysconfig/docker-storage ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker-storage "${info_system}"/docker/docker-sysconfig-storage
+    ok
+  else
+    info "/etc/sysconfig/docker-storage not found"
+  fi
+}
+
+
+get_docker_daemon_json(){
+  try "Gathering Docker daemon.json"
+
+  if [ -e /etc/docker/daemon.json ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/docker/daemon.json "${info_system}"/docker/daemon.json
+    ok
+  else
+    info "/etc/docker/daemon.json not found"
+  fi
+}
+
+get_docker_systemd_config(){
+  try "Gather Docker Daemon Service file"
+  
+  if systemctl cat docker.service > "${info_system}"/docker/docker.service 2>/dev/null; then
```

Comment:
In the same manner you can add a second branch to collect the `containerd.service` - which I suspect users are less likely to configure. Still, it might be helpful to have.

Reply:
Added contaienrd.service collection 👍